index.js

```diff
@@ -0,0 +1,76 @@
+function sayHello(name) {
+    console.log("Hello, " + name + "!");{
+    if (name="Mark")
+    console.log("Hi, " + name + "!");}
+
+
+}
+  
+sayHello("Oleg");
+sayHello("Viktor");
+sayHello("Mark");
+
+const n = [3];
+const m = [4];
```

Comment:
what the purpose of storing this numbers as an array? 
The right way: 
`const n = 3`
`const m = 4`